### PR TITLE
All images in Section 7.1

### PR DESCRIPTION
**Pull request by supervladimir**
Image descriptions for all images in Section 7.1.

### CODE REVIEW

File: ptx/sec_ABC.ptx [hunk 1]
```diff
@@ -63,7 +63,17 @@
         <caption/>
       <!-- START figures/fig_abcintroa.tex -->
         <image xml:id="img_abcintroa">
-          <description></description>
+          <description>
+            <p>
+              Graph of two functions <m> f(x) </m> and <m> g(x) </m> lying on the x-y plane.
```
Comment (sean-fitzpatrick) [inline ptx/sec_ABC.ptx]: Use `<m>xy</m>` plane for consistency with other parts of the book.

File: ptx/sec_ABC.ptx [hunk 2]
```diff
@@ -95,7 +105,13 @@
         <caption/>
       <!-- START figures/fig_abcintrob.tex -->
         <image xml:id="img_abcintrob">
-          <description></description>
+          <description>
+            <p>
+              Graph of the same two functions <m> f(x) </m> and <m> g(x) </m> as in the previous image.
+              Now, the area lying between the marked points a and b and below <m> f(x) </m> and above <m> g(x) </m> is subdivided into 10 exact verticle slices, whose areas added together give the exact area of this region.
```
Comment (sean-fitzpatrick) [inline ptx/sec_ABC.ptx]: Typo: verticle should be vertical

File: ptx/sec_ABC.ptx [hunk 3]
```diff
@@ -130,7 +146,14 @@
         <caption/>
       <!-- START figures/fig_abcintroc.tex -->
         <image xml:id="img_abcintroc">
-          <description></description>
+          <description>            
+            <p>
+              Graph of the same two functions <m> f(x) </m> and <m> g(x) </m> as in the previous two images. 
+              Now, the area lying between the marked points a and b and below <m> f(x) </m> and above <m> g(x) </m> is subdivided into 10 approximate verticle slices.
```
Comment (sean-fitzpatrick) [inline ptx/sec_ABC.ptx]: Use `<m>a</m>` and `<m>b</m>`

File: ptx/sec_ABC.ptx [hunk 4]
```diff
@@ -625,7 +702,25 @@
...
+              <description>
+                <p>
+                  A graph of the previous sketch of a lake with vertically strenching length measurements.
+                  The curve begins at the y-axis at the point <m>(0,3)</m>.
+                  From this point, the curve slopes upwards until it reaches a peak near the point <m>(6,7)</m>.
+                  After reaching a peak, the curve slopes downwards towards the point <m>(12,3)</m>.
+                  From the point <m>(0,3)</m>, the downward portion of the curve reaches its minimum near <m>(7,1)</m>, after which it goes upwards to meet at the point <m>(12,3)</m>.
+                </p>
+                <p>
+                  There are 5 verticle length measurements given which occur at all even x starting at <m>x=2</m>.
```
Comment (sean-fitzpatrick) [inline ptx/sec_ABC.ptx]: spelling of vertical again

**Review by sean-fitzpatrick — COMMENTED** [review]
This is an excellent start! (And by far the easiest review for me.)
Mostly very minimal issues. Run a find/replace to change 'verticle' to 'vertical' and you're mostly good to go.

**Comment by supervladimir** [comment]
The errors for this section should now be fixed.

**Comment by sean-fitzpatrick** [comment]
Great! You have the first pull request to be successfully merged.

**Pull request merged**